src/lwc/geFormField/geFormField.js

```diff
@@ -384,19 +405,70 @@ export default class GeFormField extends LightningElement {
 
         }
 
-        lookup.setSelected({value, displayValue});
-
+        if (this.sourceFieldAPIName === DONATION_RECORD_TYPE_NAME.fieldApiName) {
```

Comment:
I know you have a similar comment up above, but maybe just include a similar comment here about handling the Donation Record Type Name as a special case

Reply:
Good call - I added a comment to explain the special case for RecordTypeId fields, and also made a change to get the RecordType Name from the stored objectInfo property in this component which should improve performance by preventing the geFormFieldLookup component from having to query for the name.